argument_parser.py

```diff
@@ -60,5 +72,14 @@ def get_parameters():
                         The correct format should be YYYY-MM-DD""",
                         default=current_date)
 
+    parser.add_argument('-temp', metavar='temperature', type=str, choices=['C', 'F'],
+                        help="""Select the Celsius [C] or Fahrenheit [F] temperature
+                        scale (default: C).""", default='C')
+
+    parser.add_argument('-weather_components', metavar='weather_components',
+                        type=str, choices=['wind', 'pressure'], help="""Select
```

Comment:
The `type=str` makes this parameter a one-to-pick option. The documentation states:
>Additional weather components to choose from, e.g.: ‘wind’, ‘pressure’, …

..which makes the impression we should be able to pick one or more components here, therefore I believe this should be a list parameter. And again, for the choices, why we don't reuse the already existing variable here?

Reply:
@KrzysiekJodlowski 
It is not so trivial. Changing the type from 'str' to 'list' does not help when you have an argument with choices. You would need to have lists to choose from. So what I did, for now, is leave type= str and add one option to choices = ['wind', 'pressure', 'both']. Other options would be creating two arguments: pressure and wind, with options yes/no.

Reply:
@deerhunter4 got it, yes, as we discussed separately this can be solved in two ways:
- either making it a list parameter and checking it only with a custom code, leaving out the `choices` since it accepts only one option from the list
- making those separate params, although that means more code and also updating the FD accordingly

I'm fine with both options and also with an alternative to have them handled in a later phase. Please think about it and let me know, though, at first glance, I think we should have them and have them as a list.

Reply:
@KrzysiekJodlowski 
So I finally decided to make separate parameters from weather_components. And because it is now a 'flag' parameter we do not need any extra tests for them (at least I think so). I also corrected README.md file and FD file.
Additionally, I want to point out that actually there was a possibility to make parameter in type list. Check 'append' for acion [https://docs.python.org/3/library/argparse.html#action](url) . Still, I think that making two separated parameters is better idem.

Reply:
@deerhunter4 got it, thanks for updating this, this was one of the options to go with, and using flag params simplifies that part - good idea!